testutils/channel.go

```diff
@@ -59,7 +60,7 @@ func NewServerChannel(opts *ChannelOpts) (*tchannel.Channel, error) {
 	return ch, nil
 }
 
-var totalClients uint32
+var totalClients = atomic.Uint32{}
```

Comment:
sorry for the nits, but we should just do
```
var totalClients atomic.Uint32
```
i want to avoid seeing `atomic.Uint32{}` (which is one of the main reasons to avoid the pointer syntax)

can you replace all instances of `atomic.Uint32{}` in the diff.